evm/deploy-contract.js

```diff
@@ -113,7 +113,13 @@ async function getConstructorArgs(contractName, config) {
         }
 
         case 'Operators': {
-            return [];
+            const owner = contractConfig.owner;
```

Comment:
if owner is not set, use the deployer address instead, and set the field in the contract config